wallet/transaction.go

```diff
@@ -913,26 +913,32 @@ func (txMint txPlutusTokenMintInputs) Apply(args *[]string) error {
 	}
 
 	parts := make([]string, 0, len(txMint.tokens))
+	populated := false
 
 	for _, token := range txMint.tokens {
 		sign := ""
 		if token.IsNegative {
 			sign = "-"
 		}
 
-		parts = append(parts, fmt.Sprintf("%s%d %s", sign, token.Amount, token.String()))
+		if token.Amount > 0 {
+			parts = append(parts, fmt.Sprintf("%s%d %s", sign, token.Amount, token.String()))
+			populated = true
+		}
 	}
 
-	mintArgs := []string{
-		"--mint", strings.Join(parts, " + "),
-		"--mint-tx-in-reference", txMint.txInReference.String(),
-		"--mint-plutus-script-v2",
-		"--mint-reference-tx-in-redeemer-value", "0",
-		"--mint-reference-tx-in-execution-units", fmt.Sprintf("(%d,%d)", txMint.CPU, txMint.Memory),
-		"--policy-id", txMint.tokensPolicyID,
-	}
+	if populated {
```

Comment:
just a question: should we throw error "invalid data" or similar if not populated?